tuxemon/core/event/actions/clear_variable.py

```diff
@@ -0,0 +1,56 @@
+#
+# Tuxemon
+# Copyright (c) 2020      William Edwards <shadowapex@gmail.com>,
+#                         Benjamin Bean <superman2k5@gmail.com>
+#
+# This file is part of Tuxemon
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Contributor(s):
+#
+# Adam Chevalier <chevalierAdam2@gmail.com>
+
+from tuxemon.core.event.eventaction import EventAction
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+# noinspection PyAttributeOutsideInit
+class ClearVariableAction(EventAction):
+    """ Clears the value of var from the game.
+
+    Valid Parameters: string variable_name
+
+    **Examples:**
+
+    >>> EventAction.__dict__
```

Comment:
Just a FYI, Im strongly considering removing all these "examples of dictionary" typ docstrings because they get out of date and don't offer much, imo.  you can leave it now, but there will be a discussion eventually about them